source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -1,20 +1,29 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as containers from "./containers";
+import { DeviceEmulation } from "./device-emulation";
 
 export * from "./adaptivecards-designer-standalone";
 export * from "./containers/index";
 
 export const defaultMicrosoftHosts: containers.HostContainer[] = [
-	new containers.WebChatContainer("Bot Framework WebChat", "containers/webchat-container.css"),
-	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
-	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
-	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
+	new containers.WebChatContainer("Bot Framework WebChat", "containers/webchat-container.css", ),
+	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css", true),
+	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css", true),
+	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css", true),
 	new containers.VivaConnectionsContainer("Viva Connections", "containers/viva-connections-container.css"),
-	new containers.DarkCortanaContainer("Cortana Skills - Dark", "containers/cortana-container-dark.css"),
-	new containers.LightCortanaContainer("Cortana Skills - Light", "containers/cortana-container-light.css"),
-	new containers.BotFrameworkContainer("Bot Framework Other Channels (Image render)", "containers/bf-image-container.css"),
+	new containers.DarkCortanaContainer("Cortana Skills - Dark", "containers/cortana-container-dark.css", true),
+	new containers.LightCortanaContainer("Cortana Skills - Light", "containers/cortana-container-light.css", true),
+	new containers.BotFrameworkContainer("Bot Framework Other Channels (Image render)", "containers/bf-image-container.css", true),
 	new containers.CortanaClassicContainer("Cortana Skills (Classic)", "containers/cortana-classic-container.css"),
-	new containers.FederatedSearchContainer("Federated Search","containers/federated-search-container.css"),
+	new containers.FederatedSearchContainer("Federated Search","containers/federated-search-container.css", true),
 	...containers.BerlinContainer.allContainers(),
```

Comment:
@JeanRoca You can see here which hosts are opted into device emulation (`true` as the third argument opts into device emulation). I assigned them based on whether their CSS rules specify a width for the card container.

Please let me know if any of this should change.